labeling/codebase/localcodebase_test.go

```diff
@@ -153,3 +155,62 @@ func TestLocalCodebase_ReadFile(t *testing.T) {
 		})
 	}
 }
+
+func TestLocalCodebase_files(t *testing.T) {
+	t.Run("doesn't error", func(t *testing.T) {
+		_, err := LocalCodebase{BasePath: "../.."}.files()
+		if err != nil {
+			t.Errorf("got %v, expected no error", err)
+			return
+		}
+	})
+
+	t.Run("README.md (at the root) comes before this file (localcodebase_test.go)", func(t *testing.T) {
+		gotFiles, _ := LocalCodebase{BasePath: "../.."}.files()
+		thisFilePos := -1
+		readmePos := -1
+		for i, f := range gotFiles {
+			if f == "README.md" {
+				readmePos = i
+			} else if filepath.Base(f) == "localcodebase_test.go" {
+				thisFilePos = i
+			}
+		}
+		if thisFilePos == -1 {
+			t.Error("this file not found")
+		}
+		if readmePos == -1 {
+			t.Error("README.md file not found")
+		}
+		fmt.Printf("files found at positions (%d, %d)\n", readmePos, thisFilePos)
+		if readmePos > thisFilePos {
+			t.Error("expected the order to be ascending")
+		}
+	})
+
+	t.Run("finds testdata contents", func(t *testing.T) {
+		gotFiles, _ := LocalCodebase{BasePath: "../..", maxDepth: 3}.files()
+
+		for _, f := range gotFiles {
+			if filepath.Base(f) == "find.me" {
+				fmt.Println("find.me file found")
```

Comment:
Perhaps just a comment to keep test output clean?

Reply:
Good call. Fixed.